pkg/logs/auditor/auditor.go

```diff
@@ -193,6 +193,11 @@ func (a *Auditor) updateRegistry(identifier string, offset string) {
 		// specially want to avoid storing the offset
 		return
 	}
+	if _, err := time.Parse(config.DateFormat, offset); err != nil {
```

Comment:
I think offsets aren't necessarily strings in the datetime format, they can also be integers (as strings in base 10), see for example in the file tailer: https://github.com/DataDog/datadog-agent/blob/c37f3341f3cc178351a8d72e31706aca75bfd7e5/pkg/logs/input/file/tailer.go#L239

So, this logic should probably live in the docker tailer? (i.e. add some logic in the tailer to validate that the offset is valid, and then only update the registry here if the offset is indeed valid. Not sure if we can use an empty identifier to convey that meaning or something else, as I haven't looked what the identifier is used for elsewhere in the code)

Reply:
(nevermind, this is addressed by Alexandre above)

Reply:
yes I will move it in the tailer